src/Builder/ResponseBuilder.php

```diff
@@ -48,38 +48,13 @@ public function getResponse()
      */
     public function setHeadersFromArray(array $headers)
     {
-        $statusLine = trim(array_shift($headers));
-        $parts = explode(' ', $statusLine, 3);
-        if (count($parts) < 2 || substr(strtolower($parts[0]), 0, 5) !== 'http/') {
-            throw new \UnexpectedValueException(
-                sprintf('"%s" is not a valid HTTP status line', $statusLine)
-            );
-        }
-
-        $reasonPhrase = count($parts) > 2 ? $parts[2] : '';
-        $this->response = $this->response
-            ->withStatus((int) $parts[1], $reasonPhrase)
-            ->withProtocolVersion(substr($parts[0], 5));
-
```

Comment:
I dont think this should be handled via the addHeaderLine as it's not a header but more the status string, also now a request like this : 

```
Header: headervalue
GET / HTTP/1.1
```

would be correctly parsed but his invalid and should throw an error